packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -560,6 +576,71 @@ class ScrollableBaseNative extends Component {
 		}
 	}
 
+	updateFocusAfterVoiceControl = () => {
+		const spotItem = Spotlight.getCurrent();
+		if (spotItem && this.uiRef.containerRef.contains(spotItem)) {
+			const
+				viewportBounds = this.uiRef.containerRef.getBoundingClientRect(),
+				spotItemBounds = spotItem.getBoundingClientRect(),
+				nodes = this.uiRef.containerRef.querySelectorAll('[data-index]'),
```

Comment:
It would be better to use `spotlight.getSpottableDescendants(containerID)` instead of `this.uiRef.containerRef.querySelectorAll('[data-index]')`.

Reply:
I did change it base on your guide. Thanks~